NEWS

```diff
@@ -11,6 +11,11 @@
 ### Features:
 ### Bugfixes:
 
+## 1.16.0-rc5 (March 1, 2024)
+### Bugfixes:
+#### UCT
+* Fixed Infiniband posting error when compiler replaces SSE code with memmove()
+
```

Comment:
We haven't released RC4 yet.
This changes can be added to RC4. And it is better to make a single PR for changes in NEWS right before the creation of an RC. @shasson5 will update the NEWS for RC4 and publish the release.

Reply:
removed NEWS changes, ask was from @yosefe for combined PR, I guess we can do as you suggest.